hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieTableServiceClient.java

```diff
@@ -659,8 +662,46 @@ protected Map<String, Option<HoodiePendingRollbackInfo>> getPendingRollbackInfos
     return infoMap;
   }
 
+  /**
+   * Rolls back the failed delta commits corresponding to the indexing action.
+   * Such delta commits are identified based on the suffix `METADATA_INDEXER_TIME_SUFFIX` ("004").
+   * <p>
+   * TODO(HUDI-5733): This should be cleaned up once the proper fix of rollbacks
+   *  in the metadata table is landed.
+   *
+   * @return {@code true} if rollback happens; {@code false} otherwise.
+   */
+  protected boolean lazyRollbackFailedIndexing() {
```

Comment:
we should call out in release docs that if someone has failed indexing from previous version, they should drop that fully before moving to 0.13.0.

Reply:
minor. should we name the method "rollbackFailedIndexingInstants"

Reply:
Naming is fixed.